java/code/src/com/redhat/rhn/manager/audit/CVEAuditManagerOVAL.java

```diff
@@ -418,24 +423,33 @@ private static void syncOVALForProduct(OVALProduct product, OVALDownloader ovalD
     /**
      * Extracts OVAL metadata from the given {@code ovalFile}, clean it and save it to the database.
      * */
-    private static void extractAndSaveOVALData(OVALProduct product, File ovalFile) {
-        OvalRootType ovalRoot = new OvalParser().parse(ovalFile);
-        OVALCleaner.cleanup(ovalRoot, product.getOsFamily(), product.getOsVersion());
-        OVALCachingFactory.savePlatformsVulnerablePackages(ovalRoot);
+    private static void extractAndSaveOVALData(OVALOsProduct product, File ovalFile) {
+        OvalParser ovalParser = new OvalParser();
+        OVALResources ovalResources = ovalParser.parseResources(ovalFile);
+        ovalParser.parseDefinitionsInBulk(ovalFile, definitionsBulk -> {
+            OvalRootType ovalRoot = new OvalRootType();
```

Comment:
Looks to me that the `product.getOsFamily()` and `product.getOsVersion()` should also be use to set the `OvalRootType`.
In the method called `savePlatformsVulnerablePackages` the OS family and version are being loaded from the `OvalRootType` but looks to me that it's never set.